eng/pipelines/libraries/build-job.yml

```diff
@@ -48,10 +48,12 @@ jobs:
 
       ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
         dependsOn:
-        - ${{ if ne(parameters.osGroup, 'WebAssembly') }}:
+        - ${{ if and(ne(parameters.osGroup, 'WebAssembly'), ne(parameters.osGroup, 'iOS')) }}:
           - ${{ format('{0}_product_build_{1}{2}_{3}_{4}', parameters.runtimeFlavor, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveRuntimeBuildConfig) }}
         - ${{ if eq(parameters.osGroup, 'WebAssembly') }}:
           - ${{ format('{0}_product_build_{1}{2}_{3}_{4}', parameters.runtimeFlavor, 'linux', parameters.osSubgroup, 'x64', parameters.liveRuntimeBuildConfig) }}
+        - ${{ if eq(parameters.osGroup, 'iOS') }}:
+          - ${{ format('{0}_product_build_{1}{2}_{3}_{4}', 'mono', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveRuntimeBuildConfig) }}
```

Comment:
We could technically use parameters.runtimeFlavor here just to be consistent. Either way is fine, I'll leave it up to you.

Reply:
I see you're using runtimeFlavor == coreclr for the fan out of the iOS jobs. Do we actually use mono's S.Private.CoreLib? If not, this job should also depend on coreclr's job to finish, no? Because this will actually try to download coreclr's binaries before starting the build, right?